src/components/tools/image-tool.tsx

```diff
@@ -66,13 +61,27 @@ export default class ImageToolComponent extends BaseComponent<IProps, IState> {
 
   public componentDidMount() {
     this._isMounted = true;
+    this.syncedChanges = 0;
+    this.disposers = [];
     if (this.state.imageContentUrl) {
       this.updateImageUrl(this.state.imageContentUrl);
     }
+
+    this.disposers.push(autorun(() => {
+      const content = this.getContent();
+      if (content.changeCount > this.syncedChanges) {
+        this.setState({
+          isLoading: true,
+          imageContentUrl: content.url
+        });
+        this.syncedChanges = content.changeCount;
```

Comment:
@mklewandowski reported that drawing tool changes aren't synced between tiles immediately. I suspect that a call to `this.forceUpdate()` might be required here, analogous to the one in `table-tool.tsx`.

Reply:
I think you meant for this comment to be in `drawing-layer.tsx`, but initial testing seems to show that it works.

Reply:
🤦‍♂ Yes, I did. Thx for understanding where it was meant to go.

Reply:
Fixed.